.github/ISSUE_TEMPLATE/team-update.md

```diff
@@ -1,21 +1,47 @@
 ---
-name: Team updates
-about: Giving a team update
-title: "Tech Team Update: [DATE]"
-labels: team-process
+name: "🙌 Team sync"
+about: Syncronize the team's goals and actions
+labels: team-sync
+title: "Team Sync - {{ date | date('dddd, MMMM Do') }}"
 ---
 
-Hey @2i2c-org/tech-team - time to fill in some updates about what you've been up to the last couple of weeks!
-Can folks fill out the [HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw) with their own updates? ✨ 
+This is a @2i2c-org/tech-team sync 🎉🎉🎉! This is a way for 2i2c Team Members to triage work, point out issues that require attention, and make sure we are working well as a team. This issue will be closed in 24 hours.
 
-- **Updates HackMD**: https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw
-- **Team Sync history**: https://2i2c.org/team-compass/team/tech/sync/
+### Team Goals
 
-# ToDo
+See the [Team Goal label](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Agoal) for a list of team goals that we are currently working towards.
 
-- [ ] Clean up the [HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw) for this update
-- [ ] Ping the team members in [`#tech-updates`](https://2i2c.slack.com/archives/C01GLCC1VCN)
-- [ ] Wait 2-3 days
-- [ ] Copy/paste into the `team-compass` repository
-- [ ] Clean up the HackMD
-- [ ] Link to new updates in `team-compass/` in [`#tech-updates`](https://2i2c.slack.com/archives/C01GLCC1VCN)
+### Issues Requiring attention
+
+These issues require attention quickly:
+
+- **New Hubs**: The [Needs Hub issues](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Needs+Hub%22) are new hubs that need deployments.
+- **Needs Triage**: The [Needs Triage issues](https://github.com/2i2c-org/pilot/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aneeds%3Atriage) require an initial assessment and labeling.
+- **Priority Issues**: The [Priority Label issues](https://github.com/2i2c-org/pilot-hubs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority) are important and should be given attention over other issues.
+
+### Team Updates
+
+Please respond to this issue with your team update! That means anyone on the @2i2c-org/tech-team. Use the template below for your update.
+
+_Copy and paste these questions below, and answer them as you wish!_
+
+```
+**Thanks I'd like to give 🙌**
+- So-and-so helped me out a lot with XXX...
+- Thanks for Jo's work on the XXX repo...
+
+**Updates from last week ✔ **
+- I worked towards goal: <link-to-goal>
+- I merged issues XYZ
+- I had a meeting with ABC
+
+**What I'm up to next ⬜**
+- I'd like to focus on goal <link-to-goal>
+- Developing on ABC feature
+- Focusing on XXX hubs
+
+**Links to items for discussion 💬**
+- I'm have a question about goal <link-to-goal>
+- Can @XXX give a comment on issue #NN ?
+- I opened #NN for discussion, please chime in
+```
```

Comment:
I like the discussion phase towards the end.
Otherwise, you end up discussing stuff for the whole meeting and you miss the important things before that phase (thanks, update, next).